internal/config/config.go

```diff
@@ -235,5 +238,10 @@ func checks(c *config) error {
 		return err
 	}
 
+	branchRe := regexp.MustCompile("^[a-z0-9-_]+$")
+	if !branchRe.MatchString(c.GitDefaultBranch) {
+		return fmt.Errorf("invalid git default branch name: %s", c.GitDefaultBranch)
+	}
+
```

Comment:
I tend to prefer having input validation, but I suppose it would throw errors if an invalid branch name is specified, so removing this is probably fine.